miniapps/electromagnetics/makefile

```diff
@@ -59,10 +59,10 @@ include $(MFEM_TEST_MK)
 RUN_MPI = $(MFEM_MPIEXEC) $(MFEM_MPIEXEC_NP) $(MFEM_MPI_NP)
 volta-test-par: volta
 	@$(call mfem-test,$<, $(RUN_MPI), Electromagnetic miniapp,\
-	-maxit 2 -dbcs 1 -dbcg -ds '0.0 0.0 0.0 0.2 8.0')
```

Comment:
What's the reason for this change? If I remember correctly single quotes were necessary for srun on LC...

Reply:
It was failing on rzalastor with single quotes.  I sure hope we're not in a situation where we need different quotes on different platforms.

Reply:
@aaroncblack, do you remember the issue/reason for the quotes?